bedrock/settings/base.py

```diff
@@ -1265,6 +1265,7 @@ def lazy_email_backend():
     "'unsafe-inline'",
 )
 CSP_CHILD_SRC = (
+    'stomlinson.dev.lcip.org',
```

Comment:
Rather than include this in the list that gets used for production, the newly opened #4581 may be a better approach here (thanks @jpetto!)